x-pack/plugins/ml/public/explorer/explorer_controller.js

```diff
@@ -18,6 +18,7 @@ import '../components/annotations/annotations_table';
 import '../components/anomalies_table';
 import '../components/controls';
 import '../components/job_select_list';
+import '../components/job_selector'; // TODO: Do we need this?
```

Comment:
Did you find out the answer to this `TODO` !?

Reply:
Ah nice catch - removed it as it's not needed. 👍